pkg/dataMover/data_mover.go

```diff
@@ -113,8 +131,25 @@ func (this *DataMover) CopyToRepo(peID astrolabe.ProtectedEntityID) (astrolabe.P
 }
 
 func (this *DataMover) CopyFromRepo(peID astrolabe.ProtectedEntityID) (astrolabe.ProtectedEntityID, error) {
+	backupRepository := builder.ForBackupRepository("Not use").Result()
+	return this.copyFromRepo(peID, backupRepository)
+}
+
+func (this *DataMover) CopyFromRepoWithBackupRepository(peID astrolabe.ProtectedEntityID, backupRepository *backupdriverv1.BackupRepository) (astrolabe.ProtectedEntityID, error) {
+	this.Info("CopyFromRepoWithBackupRepository was called")
+	return this.copyFromRepo(peID, backupRepository)
+}
+
+func (this *DataMover) copyFromRepo(peID astrolabe.ProtectedEntityID, backupRepository *backupdriverv1.BackupRepository) (astrolabe.ProtectedEntityID, error) {
 	log := this.WithField("Remote PEID", peID.String())
 	log.Infof("Copying the snapshot from remote repository to local.")
+	if backupRepository.Name != "Not use" {
```

Comment:
If Name is set to "Not use', what will the setting for S3PETM be?  How will we avoid a NPE at line 119?